src/client/client.ts

```diff
@@ -80,7 +80,7 @@ class ClientRequestImpl {
     let methodUpperCase = this.method.toUpperCase()
 
     const headerValues: Record<string, string> = {
-      ...(args?.header ?? {}),
+      ...args?.header,
```

Comment:
```typescript
const a = undefined
{ ...a, b: 1 }
// { b: 1}
```

Reply:
I put out a PR related to this.

https://github.com/exoego/hono/pull/5